File: Algorithms_Data_Structures/Library_Management_System/LibraryTest.java
import java.util.Arrays;

public class LibraryTest {
    public static void main(String[] args) {
        Book[] catalog = {
            new Book("B1", "Java Core", "Horstmann"),
            new Book("B2", "Clean Code", "Robert Martin"),
            new Book("B3", "Algorithms", "Sedgewick")
        };

        System.out.println("Linear Search Lookup:");
        Book b1 = LibrarySearch.linearSearch(catalog, "Clean Code");
        System.out.println("Result: " + b1);
        Arrays.sort(catalog);

        System.out.println("\nBinary Search Lookup:");
        Book b2 = LibrarySearch.binarySearch(catalog, "Algorithms");
        System.out.println("Result: " + b2);
    }
}

File: Week_1/Algorithms_Data_Structures/Library_Management_System/LibraryTest.cs
using System;

public class LibraryTest {
    public static void Main(string[] args) {
        Book[] catalog = {
            new Book("B1", "Java Core", "Horstmann"),
            new Book("B2", "Clean Code", "Robert Martin"),
            new Book("B3", "Algorithms", "Sedgewick")
        };

        Console.WriteLine("Linear Search Lookup:");
        Book b1 = LibrarySearch.LinearSearch(catalog, "Clean Code");
        Console.WriteLine("Result: " + b1);
        Array.Sort(catalog);
        Console.WriteLine("\nBinary Search Lookup:");
        Book b2 = LibrarySearch.BinarySearch(catalog, "Algorithms");
        Console.WriteLine("Result: " + b2);
    }
}